release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -565,6 +565,25 @@ To manage the cloud controller manager and cloud node manager deployments and li
 
 For more information, see the xref:../operators/operator-reference.adoc#cluster-cloud-controller-manager-operator_cluster-operators-ref[Cluster Cloud Controller Manager Operator] entry in the _Cluster Operators reference_.
 
+[id="ocp-4-15-planned-psa-restricted-enforcement"]
+=== Future restricted enforcement for pod security admission
+
+Currently, pod security violations are shown as warnings and logged in the audit logs, but do not cause the pod to be rejected.
```

Comment:
```suggestion
Currently, pod security violations are shown as warnings in the audit logs without resulting in the rejection of the pod.
```

Reply:
Works for me! Updated - please take another look and let me know how it looks, thanks!